app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -86,18 +86,22 @@ form.block-form {
       text-align: center;
       color: $text-grey;
       font-size: 20px;
+      &.mail {
+        font-size: 19px;
+        top: 7px;
+      }
     }
 
     .entypo:nth-of-type(2) {
-      top: 50px;
+      top: 47px;
     }
 
     .entypo:nth-of-type(3) {
-      top: 90px;
+      top: 87px;
     }
 
     .entypo:nth-of-type(4) {
```

Comment:
The fourth item. Icons are placed from the top so it's the cleanest solution I found.